akka-docs/src/test/java/jdocs/remoting/RemoteDeploymentDocTest.java

```diff
@@ -4,24 +4,20 @@
 
 package jdocs.remoting;
 
+import akka.actor.*;
 import akka.testkit.AkkaJUnitActorSystemResource;
+import akka.testkit.AkkaSpec;
 import jdocs.AbstractJavaTest;
 import org.junit.ClassRule;
 import org.junit.Test;
 
 import com.typesafe.config.ConfigFactory;
 
 // #import
-import akka.actor.ActorRef;
```

Comment:
These imports are useful to include in the documentation snippets generated from this source file, could you put them back?

Reply:
ok